index.html

```diff
@@ -3,10 +3,19 @@
   <head>
     <link rel="stylesheet" href="css/index.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
-    <img src="IMG_0891.jpg" width="300" height="500" />
+   
     <title>About Perry Littrell</title>
+    <nav class="menu">
+      <ul>
+        <li><a href="#about">About</a></li>
+        <li><a href="#experience">Experience</a></li>
+        <li><a href="#skills">Skills</a></li>
```

Comment:
I noticed while looking at your website, that some of these nav links (`about` and `experience`) go to the `<p>` element, while `skills` go to the header `<h2>` of the section. Specifically, some of these ids are in the `<p>` element, which is awkward navigation-wise, while the `skills` id goes to the header element, which is more what I would expect. Could we move the id from the `<p>` element to the `<section>` element, for consistency?

Reply:
hi C! thank you for the feedback. I'm trying to work through it now.